### PR TITLE
Update chain.json

### PR DESCRIPTION
**Pull request by nodeiistt**

### CODE REVIEW

File: arkh/chain.json [hunk 1]
```diff
@@ -49,6 +49,10 @@
         "address": "https://asc-dataseed.arkhadian.com/",
         "provider": "arkhnetwork"
       },
+      {
+        "address": "https://rpc-arkh.nodeist.net/",
+        "provider": "arkhnetwork"
```
Comment (JeremyParish69) [inline arkh/chain.json:54]: Should the providers be 'Nodeist'?

Comment (dylanschultzie) [inline arkh/chain.json:54]: ⬆️

**Review by dylanschultzie — CHANGES_REQUESTED** [review]

**Comment by dylanschultzie** [comment]
@nodeiistt please provide better PR names in the future

File: arkh/chain.json (hunk 1 above)
Comment (nodeiistt) [reply arkh/chain.json:54]: I'm opening a new pr thanks.

**Pull request closed without merging**